src/components/control/autocompleter.jsx

```diff
@@ -41,7 +43,12 @@ const Autocompleter = forwardRef(function Autocompleter(props, ref) {
     executeModels(apiEndpoint(props.searchParam, searchString))
       .then(({ json, mounted }) => {
         if (mounted) {
-          setModels(json.results.map(result => json[result.key][result.id]));
+          const jsonResults = json.results.map(result => json[result.key][result.id]);
```

Comment:
What is the difference between search models and models? They seem to both represent API data.

Reply:
This is to fix a bug where the dropdown list would only include previous search results. We need to manage two lists so that you can open the dropdown when you are searching vs. not searching and see the correct set.

Reply:
This seems like an indirect way to fix the bug. Usually, as a React pattern, the component will perform effects after state changes. I would prefer to see a solution that would re-fetch the correct data after the user is done searching. Keeping two lists just seems like an improper modeling of the data fetching. 

If the logic is simplified, I would prefer to do that change in this PR. Otherwise, let's keep the two lists.